java-vtl-script/src/test/java/no/ssb/vtl/script/functions/VTLSubstrTest.java

```diff
@@ -0,0 +1,163 @@
+package no.ssb.vtl.script.functions;
+
+/*
+ * ========================LICENSE_START=================================
+ * Java VTL
+ * %%
+ * Copyright (C) 2016 - 2017 Pawel Buczek
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =========================LICENSE_END==================================
+ */
+
+import no.ssb.vtl.model.VTLNumber;
+import no.ssb.vtl.model.VTLObject;
+import no.ssb.vtl.model.VTLString;
+import org.assertj.core.util.Lists;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.*;
+
+
+public class VTLSubstrTest implements VTLFunctionTest {
+
+    private VTLSubstr vtlSubstr;
+
+    @Before
+    public void setUp() throws Exception {
+        vtlSubstr = VTLSubstr.getInstance();
+    }
+
+    @Test
+    public void testInvokeWithPositiveArgs() throws Exception {
```

Comment:
I still think this test method needs to be broken up.
And since the test name is inherited from the VTLNumberFunctionTest interface it does not really make sense here (even with the little name change). That the `startPosition` and `length` arguments are positive integers does not fully describe the _concepts_ being tested here